packages/node/src/yargs.ts

```diff
@@ -22,6 +22,21 @@ export const yargsOptions = yargs(hideBin(process.argv))
       return forceCleanInit();
     },
   })
+  .command({
+    command: 'reindex',
+    handler: (argv) => {
+      initLogger(
+        argv.debug as boolean,
+        argv.outputFormat as 'json' | 'colored',
+        argv.logLevel as string | undefined,
+      );
+
+      // lazy import to make sure logger is instantiated before all other services
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const { reindexInit } = require('./subcommands/reindex.init');
+      return reindexInit(argv._[1]);
```

Comment:
We can do better for getting the block height from the args. Look at the yargs docs